docs/reference/contributing/target/Watchdog.md

```diff
@@ -1,25 +1,45 @@
-<h3 id="watchdog-port">Watchdog</h3>
+### Watchdog
 
-[Include a brief description here.]
+The Watchdog HAL API provides a low-level interface to the Independent Watchdog Timer of a target. Implementing the Watchdog API is not mandatory, but by implementing it, you can use the hardware watchdog timer to detect and recover from computer malfunctions.
```

Comment:
Query: Is Independent Watchdog Timer an API? If not, what is it? Should it be capitalized?

Reply:
The Independent Watchdog Timer is the name all the vendors give to the simple/default watchdog timer in their documentation. It's usually to contrast if with the Window Watchdog Timer, but because the Window mode is not supported in the HAL API I guess the *Independent* part isn't required.